main/zoe/api/zoe.md

```diff
@@ -121,6 +148,15 @@ const installationP = await E(zoe).install(bundle);
 - Returns: `{Promise<Instance>}`
 
 Returns a `Promise` for the contract `instance` the `invitation` is part of.
+
+An `Instance` record has six properties: 
```

Comment:
InstanceRecord is gone. The instance object is opaque. (it's a handle without any properties other than identity.)  It represents a particular installed instance of a contract.

It can be obtained from calling `E(zoe).startInstance()` or from an invitation. 
It is a required parameter in `getBrands()`, `getTerms()`, `getIssuers()`, `getPublicFacet()`.